src/main/java/duke/processors/TaskList.java

```diff
@@ -0,0 +1,90 @@
+package duke.processors;
+
+import duke.task.Task;
+import java.util.ArrayList;
+
+/**
+ * This class is used to store Task class in a ArrayList and perform
+ * operations on the list.
+ */
+public class TaskList {
+
+    private final ArrayList<Task> TASKS = new ArrayList<>();
+    private int count;
+
+    /**
+     * A constructor for TaskList class.
+     * @param duke FileHandler to read data from txt file
+     */
+    public TaskList(FileHandler duke) {
+        duke.readFile(TASKS);
+        count = TASKS.size();
+    }
+
+    /**
+     * Return the Task at the index given.
+     * @param ind index
+     * @return Task
+     */
+    public Task get(int ind) {
+
+        return this.TASKS.get(ind);
+    }
+
+    /**
+     * Add a task to the ArrayList.
+     * @param task the task to be added
+     */
+    public void addTasks(Task task) {
```

Comment:
Consider making the method name "addTasks" singular since it adds only one task at a time?